.github/workflows/conformance-aks-v1.11.yaml

```diff
@@ -178,7 +178,7 @@ jobs:
       - name: Checkout PR code
```

Comment:
[Nit] The step title is now incorrect :D

Reply:
Fixed